.github/workflows/deploy-ghpages-latest-stable.yml

```
@@ -109,7 +116,7 @@ jobs:
           else
             missing_doc="latest"
         fi
-        URL="https://qibogang.github.io/qibo/${missing_doc}.tar.gz"
+        URL="${{ inputs.url }}/${{ inputs.docs-folder }}/${missing_doc}.tar.gz"
```

You did not propagate the change:
```suggestion
        URL="${{ inputs.url }}/${{ inputs.project }}/${missing_doc}.tar.gz"
```